### PR TITLE
add logic to handle sdk-social-aa-connector

### PR DESCRIPTION
**Pull request by on47sky**

### CODE REVIEW

File: src/hooks/useAaConnector.ts [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+
+let useAccountWithAA: typeof useAccountFn = () => useContext(DumbContext) as any
+let ready = false
+
+export const useExtendedAccount: typeof useAccountFn = () => {
+  const account = useAccount()
+  const accountWithAA = useAccountWithAA()
+
+  return ready ? accountWithAA || { ...account, isAAWallet: false, ready } : { ...account, isAAWallet: false, ready }
```
Comment (CleanBread) [inline src/hooks/useAaConnector.ts]: I don't get it) why not just `accountWithAA || { ...account, isAAWallet: false, ready }`?

File: src/contexts/chain/config.ts [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+
+export type ChainContextValue = Omit<ChainData, 'chain'> & {
+  appChain: Omit<Chain, 'id'> & { id: ChainId }
+  walletChain: Chain | undefined
+  isRightNetwork: boolean
+  setAppChainId: (chainId: ChainId) => void
+}
+
+export const ChainContext = createContext<ChainContextValue | null>(null)
```
Comment (CleanBread) [inline src/contexts/chain/config.ts]: can't it be imported from chain context without creating addition file?

File: src/hooks/useAaConnector.ts [hunk 3]
```diff
@@ -0,0 +1,35 @@
+import type { useAccount as useAccountFn, useAAWalletClient as useAAWalletClientFn } from '@azuro-org/sdk-social-aa-connector'
+import { useAccount } from 'wagmi'
+import { useContext } from 'react'
+import { ChainContext as DumbContext } from '../contexts/chain/config'
```
Comment (CleanBread) [inline src/hooks/useAaConnector.ts]: just `const DumbContext = createContext(null)` ?

File: src/hooks/useAaConnector.ts [hunk 4]
```diff
@@ -0,0 +1,35 @@
+import type { useAccount as useAccountFn, useAAWalletClient as useAAWalletClientFn } from '@azuro-org/sdk-social-aa-connector'
```
Comment (CleanBread) [inline src/hooks/useAaConnector.ts:2]: mb useAccount from sdk connector should be named as useExtendedAccount

File: src/hooks/useAaConnector.ts [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+
+export const useExtendedAccount: typeof useAccountFn = () => {
+  const account = useAccount()
+  const accountWithAA = useAccountWithAA()
+
+  return ready ? accountWithAA || { ...account, isAAWallet: false, ready } : { ...account, isAAWallet: false, ready }
+}
+
+export let useAAWalletClient: typeof useAAWalletClientFn = () => {
+  useContext(DumbContext)
```
Comment (CleanBread) [inline src/hooks/useAaConnector.ts]: ?

File: src/hooks/useAaConnector.ts (hunk 3 above)
Comment (on47sky) [reply src/hooks/useAaConnector.ts]: yep) don't know what it was 😅

Comment (on47sky) [reply src/hooks/useAaConnector.ts]: Refactored: https://github.com/Azuro-protocol/sdk/pull/47/commits/3e4e173bab30aec6eb574382cbeee358033306fb + https://github.com/Azuro-protocol/sdk/pull/47/commits/9a778231ef8a6354b0dfe06fe69cf2eb70a6ad44

File: src/hooks/useAaConnector.ts (hunk 1 above)
Comment (on47sky) [reply src/hooks/useAaConnector.ts]: it was because of not really "dumb" context, so it's fixed within refactoring

**Review by CleanBread — APPROVED** [review]

**Pull request merged**